torch/_decomp/__init__.py

```diff
@@ -578,6 +578,7 @@ def _core_aten_decompositions_post_autograd() -> (
             aten.norm,
             aten.ones,
             aten.ones_like,
+            aten.permute_copy,
```

Comment:
If you only care about inductor, can you remove this line and add it https://github.com/pytorch/pytorch/blob/main/torch/_inductor/decomposition.py#L48 ?

Reply:
yes please